frontend/src/js/controllers/teamsCtrl.js

```diff
@@ -359,6 +359,55 @@
             }, function() {
             });
         };
+
+
+        vm.editTeamName = function(ev, participantTeamId) {
+            ev.stopPropagation();
+            // Appending dialog to document.body to cover sidenav in docs app
+            var confirm = $mdDialog.prompt()
+                .title('Change Team Name')
+                .textContent('Enter the new name for the Team.')
+                .placeholder('Team Name')
+                .ariaLabel('')
+                .targetEvent(ev)
+                .ok('confirm')
+                .cancel('Cancel');
+
+            $mdDialog.show(confirm).then(function(result) {
+                var parameters = {};
+                parameters.url = 'participants/participant_team/' + participantTeamId;
+                parameters.method = 'PATCH';
+                parameters.data = {
+                    "team_name": result,
+                };
+                parameters.token = userKey;
+                parameters.callback = {
+                    onSuccess: function() {
+                        $rootScope.notify("success", "The Participant team name was successfully updated.");
```

Comment:
`The Participant team name was successfully updated.` -> `The Participant team name is successfully updated!`